<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>472</width>
    <height>254</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>471</width>
      <height>231</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>PÃ¡gina</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Usuarios</string>
     </attribute>
     <widget class="QPushButton" name="ingresar_usuarios">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>160</y>
        <width>121</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>Ingresar Usuarios</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tabla_usuarios">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>20</y>
        <width>331</width>
        <height>131</height>
       </rect>
      </property>
      <property name="rowCount">
       <number>0</number>
      </property>
      <property name="columnCount">
       <number>3</number>
      </property>
      <column/>
      <column/>
      <column/>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionIngresar_Usuarios_2">
   <property name="text">
    <string>Ingresar Usuarios</string>
   </property>
  </action>
  <action name="actionConsultar_Usuarios">
   <property name="text">
    <string>Consultar Usuarios</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
